packages/react-ui/src/theme/themeConstants.js

```diff
@@ -1,2 +1,11 @@
 // Common
 export const SPACING = 8;
+
+// Breakpoints
+export const BREAKPOINTS = {
+  XS: 320,
+  SM: 600,
+  MD: 960,
+  LG: 1280,
+  XL: 1600
+};
```

Comment:
If we have just a few, maybe, just maybe, it's better to see the values in the carto-theme.js... What do you think?
I see it a bit like too many indirection levels